python/tests/test_array.py

```diff
@@ -1875,6 +1875,17 @@ def test_to_scalar(self):
         with self.assertRaises(ValueError):
             int(a)
 
+    def test_format(self):
+        a = mx.arange(3)
+        self.assertEqual(f"{a[0]:.2f}", "0.00")
+
+        b = 0.35487
+
+        self.assertEqual(f"{b:.1f}", "0.4")
```

Comment:
I don't think this is testing anything in MLX?

Reply:
That's silly -- good catch!